packages/sit-onyx/src/components/OnyxPageLayout/examples/GridExample.vue

```diff
@@ -0,0 +1,57 @@
+<script setup lang="ts">
+import { OnyxAppLayout, OnyxButton, OnyxHeadline, OnyxPageLayout, OnyxSidebar } from "../../..";
+</script>
+
+<template>
+  <OnyxAppLayout>
+    <OnyxPageLayout>
+      <template #sidebar>
+        <OnyxSidebar label="Example sidebar" resizable>
+          <template #header> Header content </template>
+          <div class="onyx-grid-layout">
+            <div class="onyx-grid">
+              <div v-for="i in 6" :key="i" class="onyx-grid-example">
+                {{ i }}
+              </div>
+              <div class="onyx-grid-example onyx-grid-span-2">span 2</div>
+              <div class="onyx-grid-example onyx-grid-md-span-4">md span 4</div>
+              <div class="onyx-grid-example onyx-grid-span-full">full span</div>
+            </div>
+          </div>
+
+          <template #footer>
+            <OnyxButton color="neutral" label="Button" />
+            <OnyxButton label="Button" />
+          </template>
+        </OnyxSidebar>
+      </template>
+
+      <!-- page content -->
+      <OnyxHeadline is="h1">Page content</OnyxHeadline>
+      <p>
+        Breakpoints for the grid dynamically adjust based on the width of the main page content area
+        or the sidebar.
+      </p>
+
+      <div class="onyx-grid">
+        <div v-for="i in 12" :key="i" class="onyx-grid-example">
+          {{ i }}
+        </div>
+        <div class="onyx-grid-example onyx-grid-span-2">span 2</div>
+        <div class="onyx-grid-example onyx-grid-md-span-4">md span 4</div>
+        <div class="onyx-grid-example onyx-grid-span-full">full span</div>
+      </div>
+    </OnyxPageLayout>
+  </OnyxAppLayout>
+</template>
+
+<style lang="scss" scoped>
+.onyx-grid-example {
+  background-color: var(--onyx-color-component-cta-default);
```

Comment:
Jannick requested if we can make to color more subtle so they don't look like buttons :)
Also for the OnyxSidebar example